packages/js-client-grpc/proto/collections.proto

```diff
@@ -637,6 +637,7 @@ enum ReplicaState {
   Recovery = 6; // Shard is undergoing recovered by an external node; Normally rejects updates, accepts updates if force is true
   Resharding = 7; // Points are being migrated to this shard as part of scale-up resharding
   ReshardingScaleDown = 8; // Points are being migrated to this shard as part of scale-down resharding
+  ActiveRead = 9; // Active for readers, Partial for writers
```

Comment:
Protofile updates are not related to this PR. But I don't find it a problem because we're gonna add these changes into `v1.16`